### PR TITLE
switch APIs to single col layout

### PR DESCRIPTION
**Pull request by donjo**
Screenshot 2024-07-25 at 12 06 53 PM

This adds style overrides to render the APIs in a single column layout with some adjusted spacing to account for the new layout.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -272,4 +272,7 @@ body:not(:has(.ddoc)) {
       background-color: var(--color-neutral-muted) !important;
     }
   }
+  .markdown_summary :not(pre) > code {
+    background-color: var(--color-neutral-muted) !important;
```
Comment (donjo) [inline styles.css:276]: this is unrelated to the single col thing I just noticed that the style above didn't catch all code blocks

**Review by crowlKats — APPROVED** [review]
